src/ui/map.js

```diff
@@ -835,6 +837,51 @@ class Map extends Camera {
         return this;
     }
 
+    /**
+     * Adds a listener for events of a specified type.
+     *
+     * @method
+     * @name once
+     * @memberof Map
+     * @instance
+     * @param {string} type The event type to add a listen for.
+     * @param {Function} listener The function to be called when the event is fired.
+     *   The listener function is called with the data object passed to `fire`,
+     *   extended with `target` and `type` properties.
+     * @returns {Map} `this`
+     */
+
+    /**
+     * Adds a listener for events of a specified type occurring on features in a specified style layer.
```

Comment:
And this one should basically match the comment above but with the addition of `occurring on features in a specified style layer`